content/post/22-07-27-progress-report-36.md

```diff
@@ -0,0 +1,414 @@
+
+{{< newsection >}}
+## Content
+### Surpassing /tg/station 13
+*Contributed by everyone*
+
+Space Station 14 is officially being developed at a faster pace than the most active SS13 codebase, /tg/station. At the time of writing SS14 has 520 merged and 50 open pull requests (code changes) in the last 30 days, compared to 326 merged and 106 open for /tg/.
+
+And that's not even counting changes to Robust Toolbox.
+
+{{< newsection >}}
+### Gun Refactor
+*Contributed by metalgearsloth, ElectroSR, taral & probably others*
+
+Guns have been refactored in their entirety. In fact there's been so many changes to guns over the course of June that we, your benevolent Progress Report authors, lost track of them all.
+
+Here's the highlights:
+
+- Guns now have clientside prediction and are more responsive
+- Updated hitscan effects
+- Updated sprites
+- Added fire rate selectors (e.g. semi-auto and full-auto)
+- Rebalanced the weight, accuracy, and bullet spread of many guns to give different types (i.e. rifles versus SMGs) more distinction. These changes are ongoing.
+- Cleaned up gun code a lot to enable sillier and more unique guns
+- Added a development tool for visualizing bullet spread
+
+TODO: Fire selector video?
+
+{{< newsection >}}
+### Item Pricing & Piracy
+*Contributed by moony*
+
+Most things now have prices assigned allowing Cargo to, for example, sell fuel tanks for a couple hundred spacebucks. Consequentially, an increase of space piracy has been detected in the sector. 
+
+Attempts to kidnap and sell the Clown will be refused by Central Command.
+
+TODO: Pic?
+
+{{< newsection >}}
+### Drug & Drunk Shaders
+*Contributed by ElectroSR*
+
+To save costs and improve employee morale, Central Command has disabled the drug & alcohol inhibitors embedded in everyone's spinal cord, and both vices should now have much more substantial effects. Employees are reminded to avoid getting absolutely blasted while they're on the clock.
+
+Here's what the drug effects look like (Author's Note: This is not what it looks like IRL):
+
+{{< video-audio "/video/pr_36/maryjane.mp4" >}}
+
+{{< newsection >}}
+### Department Radios
+*Contributed by Morb0, ElectroSR, metalgearsloth, & notafet*
+
+Engineering finally bothered to install the rest of the station's telecommunications equipment, and the network now has sufficient bandwidth for departmental radio channels. The crew can now yell exclusively at their own department instead of trying to talk over each other on the Common frequency all the time.
+
+{{< imgw "/images/post/pr_36/radio" >}}
+
+{{< newsection >}}
+### Modular Landmines
+*Contributed by mirrorcult*
+
+The lads down in the Department of Interstellar War Crimes™ have developed a new modular landmine platform which accepts customized payloads. Casualties guaranteed or your money back! 
+
+Make sure to watch your step.
+
+{{< video-audio "/video/pr_36/mine.mp4" >}}
+
+{{< newsection >}}
+### Rat King
+*Contributed by mirrorcult & EmoGarbage404*
```

Comment:
if mirror is going to be credited under rat king, it should at least mention the thing that she actually made, the accent.

Reply:
The video is literally the accent pull request video.